iso/scripts/generate_dappnode_iso_debian.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 set -e
 
-# Source = https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/current/amd64/iso-cd/
-# updated permalink to archive of debian ISO: https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/archive/11.5.0+nonfree/amd64/iso-cd/
-ISO_NAME=firmware-11.5.0-amd64-netinst.iso
+# Source = https://cdimage.debian.org/debian-cd/12.0.0/amd64/iso-cd/debian-12.0.0-amd64-netinst.iso
+ISO_NAME=debian-12.0.0-amd64-netinst.iso
 ISO_PATH="/images/${ISO_NAME}"
-ISO_URL=https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/archive/11.5.0+nonfree/amd64/iso-cd/
-SHASUM="ce1dcd1fa272976ddc387554202013e69ecf1b02b38fba4f8c35c8b12b8f521e  ${ISO_PATH}"
+ISO_URL=https://cdimage.debian.org/debian-cd/12.0.0/amd64/iso-cd/
```

Comment:
Good to have this here:
![image](https://github.com/dappnode/DAppNode/assets/47595345/87329119-b316-47df-ac3a-a7cb2994fee6)

Reference: https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/